package.json

```diff
@@ -27,26 +27,28 @@
   "dependencies": {
     "@semantic-release/error": "^3.0.0",
     "aggregate-error": "^3.1.0",
-    "debug": "^4.3.4",
+    "debug": "^4.3.5",
     "execa": "^5.1.1",
-    "lodash": "^4.17.21",
-    "semantic-release": "^22.0.7"
+    "lodash": "^4.17.21"
```

Comment:
I was unable to update `@semantic-release/error`, `aggregate-error` or `execa` due to module errors while importing during Jest execution.